uint16 key_x
uint16 key_y
uint16 key_z
float64 reward
geometry_msgs/Vector3 normal